src/howler.core.js

```diff
@@ -1484,6 +1535,9 @@
           // Remove any event listeners.
           sounds[i]._node.removeEventListener('error', sounds[i]._errorFn, false);
           sounds[i]._node.removeEventListener(Howler._canPlayEvent, sounds[i]._loadFn, false);
+
+          // Release the Audio object back to the pool.
+          Howler._releaseHtml5Audio(sounds[i]._node);
```

Comment:
Update: added Howler.shouldMobileHtml5AudioPoolErrorWhenEmpty and a means of detecting locked Audio objects when they are released.